### ISSUE TITLE
Missing StrategyQA dataset processing and configuration code

### ISSUE THREAD

**Issue by velpro123-cell** [KANABOON1/LatentMem#4]
## Description

Thank you for releasing LatentMem. I am attempting to reproduce the experiments, particularly the StrategyQA results, but I could not find the corresponding dataset implementation and configuration in the repository.

The current code references StrategyQA in `latentmem/runner.py`:

```python
elif self.config.dataset_cfg.name == "strategyqa":
    train_size = len(self.train_dataset)
```
Moreever, I could not find the following components:
- data/strategyqa/
- configs/latentmem/strategyqa.yaml
- StrategyQA preprocessing and dataset split logic
- StrategyQA environment and answer-checking/reward implementation
- StrategyQA prompts for the supported multi-agent architectures

## Questions
1. Was the StrategyQA implementation intentionally omitted from the current release?
2. Which StrategyQA dataset source and version were used in the paper?
3. What training/validation/test splits and preprocessing steps were used?
4. Would it be possible to release the missing StrategyQA processing, environment, prompt, and configuration files?
These details would be very helpful for reproducing the reported multi-domain experiments.
Thank you for your help!
